proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -35,6 +35,9 @@ message MsgRegisterInterchainAccount {
 
 // MsgRegisterInterchainAccountResponse defines the response for Msg/RegisterAccount
 message MsgRegisterInterchainAccountResponse {
+  option (gogoproto.equal)           = false;
```

Comment:
Are the equal directives needed? See [discussion](https://github.com/cosmos/ibc-go/pull/3858#discussion_r1231248212)

Reply:
Is it ok in a remove them all in a separate PR addressing https://github.com/cosmos/ibc-go/issues/3871?

Reply:
unfortunate that we added them in the update messages since that will need to be fixed in a separate PR to 04-channel. I'd vote for removing them here if we decide to go with #3871 but not strongly opinionated on it.

Reply:
It's just much easier for me to merge this as is and then delete all the occurrences in a separate PR, instead of now looking one by one at the places where I added them and delete them. 😓

Reply:
i can also help rm the occurences in this pr!

Reply:
Thank you, @charleenfei!